Start voting after 11 minutes despite unit file.

afck opened the pull request.

afck committed:
Disable expensive LNC check.

afck committed:
Fix typo.

afck committed:
Disable LNC test.

afck committed:
Disable spurious error.

afck committed:
Disable other call sites of the expensive sees method.

Fraser999 committed:
temporarily disable failing cargo-casper test

afck committed:
Start voting after 20 minutes despite unit file.

fizyk20 reviewed (approved):
Looks like what we want right now :+1:

afck committed:
Don't validate monotonicity of endorsements.

afck committed:
11 minutes instead of 20.

afck committed:
Clear self.own_last_unit when timing out.

This prevents an edge case where we would end up adding our stored own
last unit later and potentially equivocate.

afck commented:
Closing this: It's not urgent and needs some discussion first.
